eqcorrscan/utils/catalog_to_dd.py

```diff
@@ -184,7 +183,7 @@ def _prepare_stream(stream, event, extract_len, pre_pick, seed_pick_ids=None):
         elif len(pick) == 0:
             continue
         pick = pick[0]
-        tr = stream.select(id=seed_pick_id.seed_id).merge()
+        tr = stream.select(id=seed_pick_id.seed_id)
```

Comment:
Was there a reason for removing the merge?

Reply:
I was running into some issues here, but I later reintroduced the merge with the newest commit, so please disregard this change!